controllers/expenses.js

```diff
@@ -0,0 +1,111 @@
+/* eslint-disable object-curly-newline */
+'use strict';
+
+const expensessService = require('../services/expenses.js');
+const usersService = require('../services/users.js');
+
+const getAll = (req, res) => {
+  const query = req.query;
+
+  if (!query) {
+    res.status(304).send('Not modified');
+  }
+
+  const expenses = expensessService.getAll(query);
+
+  res.status(200).send(expenses);
+};
+
+const getExpense = (req, res) => {
+  const { expenseId } = req.params;
+
+  if (isNaN(+expenseId)) {
+    res.status(400).send('Bad request');
+  }
```

Comment:
don't forget to stop the code
```suggestion
    res.status(400).send('Bad request');
    
    return;
  }
```